### PR TITLE
[OCPBUGS-45851]: Correcting etcd backup docs

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14+

Issue: https://issues.redhat.com/browse/OCPBUGS-45851

Link to docs preview: https://85926--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ etcd is the key-value store for {product-title}, which persists the state of all
 
 Back up your cluster's etcd data regularly and store in a secure location ideally outside the {product-title} environment. Do not take an etcd backup before the first certificate rotation completes, which occurs 24 hours after installation, otherwise the backup will contain expired certificates. It is also recommended to take etcd backups during non-peak usage hours because the etcd snapshot has a high I/O cost.
 
-Be sure to take an etcd backup after you update your cluster. This is important because when you restore your cluster, you must use an etcd backup that was taken from the same z-stream release. For example, an {product-title} 4.y.z cluster must use an etcd backup that was taken from 4.y.z.
+Be sure to take an etcd backup before you update your cluster. Taking a backup before you update is important because when you restore your cluster, you must use an etcd backup that was taken from the same z-stream release. For example, an {product-title} 4.y.z cluster must use an etcd backup that was taken from 4.y.z.
```
Comment (ShaunaDiaz) [inline backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc:13]: IDK, but "4.y.z" is not an example in my mind. "4.17.5" is an example. Just a thought.

**Review by ShaunaDiaz — COMMENTED** [review]
LGTM just one thought.

File: backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc (hunk 1 above)
Comment (lahinson) [reply backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc:13]: Good point! Will fix.

**Pull request merged**